ghmirror/data_structures/redis_data_structures.py

```diff
@@ -21,44 +21,55 @@
 import redis
 
 
+REDIS_ENDPOINT = os.environ.get('REDIS_ENDPOINT')
+REDIS_PORT = os.environ.get('REDIS_PORT', 6379)
+REDIS_TOKEN = os.environ.get('REDIS_TOKEN')
+
+
 class RedisCache:
     """
     Dictionary-like implementation for caching requests in Redis.
     """
     def __init__(self):
-        endpoint = os.environ.get('REDIS_ENDPOINT')
-        port = os.environ.get('REDIS_PORT', 6379)
-        password = os.environ.get('REDIS_TOKEN')
-        self.cache = redis.Redis(
-            host=endpoint,
+        write_endpoint = REDIS_ENDPOINT
+        read_endpoint = write_endpoint.replace('master', 'replica', 1)
+        port = REDIS_PORT
+        password = REDIS_TOKEN
```

Comment:
these variables are not needed anymore...

Reply:
oops. will fix.